app/controllers/staff/time_slots_controller.rb

```diff
@@ -91,4 +91,8 @@ def set_time_slot
   def time_slot_decorated
     @time_slot_decorated ||= Staff::TimeSlotDecorator.decorate(@time_slot)
   end
+
+  def json_filename
+    "#{current_event.slug}-schedule-#{DateTime.current.to_s(:db_just_date)}"
```

Comment:
The file downloads with .json on it.  Maybe when you override filename, you need to be explicit on the extension?